scripts/prepare-tests.sh

```diff
@@ -35,7 +35,7 @@ cat os.yml.fusor >> operatingsystems.yml
 rm -f hosts.yml os.yml.foreman os.yml.fusor
 ln -s ../../../katello/test/fixtures/models/* .
 
-# Create hosts file
-echo "" >> hosts.yml
-cat ../../../fusor/server/test/fixtures/hosts.yml >> hosts.yml
+# Re-link fusor hosts.yml file
```

Comment:
iirc line 35 linked a katello hosts.yml,  `echo "" >> hosts.yml` appends a newline, and then we appended our contents through the link onto katello hosts.yml (think that was a mistake on my part). Since we blow away the link and simply link ours here,  this implementation is missing the katello hosts. Probably doesn't matter for our purposes.